purchase_request/views/purchase_request_view.xml

```diff
@@ -0,0 +1,589 @@
+            </field>
+        </record>
+
+
+        <record model="ir.actions.act_window" id="purchase_request_form_action">
+            <field name="name">Purchase Requests</field>
+            <field name="type">ir.actions.act_window</field>
+            <field name="res_model">purchase.request</field>
+            <field name="view_type">form</field>
+            <field name="view_mode">tree,form</field>
+            <field name="context">{"search_default_requested_by":uid}</field>
+            <field name="search_view_id" ref="view_purchase_request_search"/>
+            <field name="help" type="html">
+              <p class="oe_view_nocontent_create">
+                Click to start a new purchase request process.
+              </p><p>
+                A purchase request is an instruction to Purchasing to procure
+                a certain quantity of materials services, so that they are
+                available at a certain point in time.
+              </p>
+            </field>
+        </record>
+
+        <menuitem
+            id="menu_purchase_request_pro_mgt"
+            parent="purchase.menu_procurement_management"
+            action="purchase_request_form_action"/>
+
+
+        <record id="purchase_request_line_tree" model="ir.ui.view">
+            <field name="name">purchase.request.line.tree</field>
+            <field name="model">purchase.request.line</field>
+            <field name="arch" type="xml">
+                <tree string="Purchase Request Lines" create="false"
+                      decoration-muted="cancelled == True">
+                    <field name="request_id"/>
+                    <field name="request_state"/>
+                    <field name="requested_by"/>
+                    <field name="assigned_to"/>
+                    <field name="date_required"/>
+                    <field name="purchased_qty"/>
+                    <field name="purchase_state"/>
+                    <field name="product_id"/>
+                    <field name="name"/>
+                    <field name="product_qty"/>
+                    <field name="product_uom_id" groups="product.group_uom"/>
+                    <field name="analytic_account_id"
+                           groups="analytic.group_analytic_accounting"/>
+                    <field name="supplier_id"/>
+                    <field name="cancelled" invisible="1"/>
+                </tree>
+            </field>
+        </record>
+        <record id="purchase_request_line_form" model="ir.ui.view">
+            <field name="name">purchase.request.line.form</field>
+            <field name="model">purchase.request.line</field>
+            <field name="priority" eval="20"/>
+            <field name="arch" type="xml">
+                <form string="Purchase Request Line" duplicate="false">
+                    <sheet>
+                        <h1>
+                            <field name="request_id"/>
+                            <field name="request_state"/>
```

Comment:
Again, this seems a weird place to have more fields apart from name, state would fit better in a status bar as in almost any other document/model.